### PR TITLE
chore: update 0x deps

### PR DESCRIPTION
**Pull request by kimpers**
# Description
This bumps all `@0x` dependencies to the latest version (which includes Lido support) and fixes issues arising from swapping to Ajv for json schemas.

Depends on 0xProject/tools#38.

# Testing Instructions

# Checklist

-   [ ] Update [documentation](https://github.com/0xProject/website/blob/development/mdx/api/index.mdx) as needed. **Website Documentation PR:**
-   [ ] Prefix PR title with `[WIP]` if necessary.
-   [ ] Add tests to cover changes as needed.
-   [x] Test changes on the staging environment with [Matcha API Staging](https://api-staging.matcha.xyz).

    -   [x] SRA/Limit orders
    -   [ ] Swap endpoints
    -   [ ] Meta transaction endpoints
    -   [x] Depth charts

    For more information see `0x API Matcha smoke test runbook` in Quip.

### CODE REVIEW

File: test/sra_test.ts [hunk 1]
```diff
@@ -314,15 +314,15 @@ describe(SUITE_NAME, () => {
...
-                        reason: 'does not match pattern "^0x[0-9a-fA-F]{40}$"',
-                    },
                     {
                         field: 'baseToken',
                         code: 1000,
-                        reason: 'requires property "baseToken"',
+                        reason: "should have required property 'baseToken'",
+                    },
+                    {
+                        field: 'quoteToken', // FIXME (xianny): bug in jsonschemas module
```
Comment (xianny) [inline test/sra_test.ts:323]: you can remove my comment since you removed the ugly `instance.` prefix!

**Review by xianny — APPROVED** [review]
looks great, thanks so much for fixing this!

**Comment by romainbutteaud** [comment]
thanks Kim, make sure you don't update `asset-swapper` and `protocol-utils` cause I already did it in https://github.com/0xProject/0x-api/pull/660/files

**Comment by kimpers** [comment]
deploy staging

**Pull request merged**